src/lib/types.ts

```diff
@@ -331,7 +331,7 @@ export namespace Tz {
     export interface Converter {
         key?: string[];
         options?: Option[] | ((definition: Definition) => Option[]);
-        endpoint?: string;
+        endpoint?: string | string[];
```

Comment:
```suggestion
        endpoints?: string[];
```

I'm thinking now, maybe always make this a `string[]`? Should simplify the z2m code: `(!c.key || c.key.includes(key)) && (!c.endpoints || c.endpoints.includes(endpointName)));`

Reply:
I fully agree with this idea to simplify `endpoints` as `string[]` only. 
Use of an array definitely makes sense as one or more `endpointName` can be associated to a given cluster. 
If I remember well, the actual `endpoint: string` is already used elsewhere : Before go further, I need to evaluate the impact.

Reply:
I don't expect it is used in many places, changing the type here and doing a `npm run build` should show you the issues.

Reply:
Thanks for the advise. I'll do that. :thumbsup:

Reply:
Finally, your expectation is verified and I found only `endpoint` used in `tuya.ts`.